lib/ruby_smb/smb2/info_type.rb

```diff
@@ -0,0 +1,21 @@
+module RubySMB
+  module SMB2
+    # Contains constants for the InfoType values as defined in
+    # [2.2.39 SMB2 SET_INFO Request](https://msdn.microsoft.com/en-us/library/cc246560.aspx) and
+    # [2.2.37 SMB2 QUERY_INFO Request](https://msdn.microsoft.com/en-us/library/cc246557.aspx)
+    module InfoType
+      # The file information
+      SMB2_0_INFO_FILE       = 0x01
+
+      # The underlying object store information
+      SMB2_0_INFO_FILESYSTEM = 0x02
+
+      # The security information
+      SMB2_0_INFO_SECURITY   = 0x01
```

Comment:
This should be 0x03

Reply:
Good catch! A bad copy/paste, thanks!

Reply:
Addressed in dcd7ff20e8253df592e5358d503d916d62fb479a